docs/pipeline/merger_rules.md

```diff
@@ -0,0 +1,5 @@
+# Merger Rules
+
+## Items
+![How items are selected as candidates and merged](../.gitbook/assets/merger_items_rule.png)
```

Comment:
I'm afraid I find this diagram very confusing - I've been staring at it for a few minutes and still don't understand it, so it's a bit tricky to say what I think would help, but some thoughts would be:

- Could you organise it so that information flows in one direction? It's not clear what's an input or an output - are the Miro and Calm works at the bottom the same type of thing as the works above them?
- I think that different arrows represent different things - eg some on the left hand side are field names in source data referring to other sources, whereas some of the ones that go left-to-right are about the results of merging (ie the data flow). Could the entity relationships vs merges/transforms be made explicit?
- Related to that - could you clarify that this also encompasses some logic in the transformers? It seems to be about more than just the merger's items rule.
- What does 'Nothing' refer to?
- Why is a SIM rule called a SIM rule? Do we use that name anywhere else? Should we be using it in code? I might also rephrase point (2) in that description to something like "Otherwise, if there is more than one item, append the items from the source to those in the target" - I'm not sure that 'verbatim' explains what happens here.

Reply:
Yea, 100%. Apologies for the slapdashness of it, this was almost meant to be a place holder for the next iteration, and a way for me to keep track of the conversations I was having with JT, so not very useful for others.

I am trying to strike a balance of keeping this PR as a stepping stone to something better. Does the description of the PR help? I'll try mimic that.

Reply:
The description is helpful, would be ideal to have something like a slightly fleshed out version of that next to an improved diagram. Possibly remove the docs from this PR and do that separately? Up to you

Reply:
Na, I'll try get it a little better than a brain splurge.

Reply:
Updated - what do you think?

The merging bit I will do post a Calm target world as it will add a bit of complexity to diagramming.

Reply:
Much clearer! Maybe change `work` -> `works` as otherwise it kinda looks like a Sierra work would be referring to itself?